src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js

```diff
@@ -232,6 +232,30 @@ define([], function() {
             return [this.options.url, delimiter, this.options.idsParameter, '=', (data || []).join(',')].join('');
         },
 
+        overlayOkCallback: function () {
+            this.sandbox.emit('husky.column-navigation.' + this.options.instanceName + '.get-marked', function (markedCollections) {
+                var data = this.sandbox.util.deepCopy(this.getData()),
+                    markedCollectionIds = markedCollections.map(function(markedCollection) {
+                        return markedCollection.id;
+                    });
+
+                markedCollections.forEach(function(markedCollection) {
+                    if ($.inArray(markedCollection.id, data) < 0) {
+                        selectLink.call(this, markedCollection);
+                    }
+                }.bind(this));
+
+                data.forEach(function (item) {
+                    if ($.inArray(item, markedCollectionIds) < 0) {
+                        this.removeHandler(item);
+                    }
+                }.bind(this));
+
+                this.sandbox.stop(getId.call(this, 'columnNavigation'));
+                this.sandbox.close($element);
```

Comment:
`$element` is not defined, and what should this actually do? Is there even a `close` method on the sandbox?